### PR TITLE
Update information about library linking

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Update README.rst and compiling.rst about
how to link the library and include development
headers

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -100,21 +100,27 @@ refer to `compiling.rst`_ or build the documentation, cf. below.
 **Linking libdrizzle-redux**::
 
 Ensure the library is in your library and include paths. For releases prior to
-version ``v6.0.2`` linking your app against libdrizzle-redux requires the flag
-``-ldrizzle-redux``::
+version ``v6.0.3`` linking your app against libdrizzle-redux requires the flag
+``-ldrizzle-redux6``::
```
Comment (ben-palmer-sociomantic) [inline README.rst]: Is this right? For versions _before_ `v6.0.3` you need to link with `-ldrizzle-redux6`?

**Review by ben-palmer-sociomantic — COMMENTED** [review]
One comment then LGTM

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: Ah true, should be `libdrizzle-redux`, thinking about v6.0.2 should be deprecated since the linking with `ldrizzle-redux6` won't work

Comment (mathias-lang-sociomantic) [reply README.rst]: I would just say `v6.x.x`, as those point release are just broken and it creates confusion. But I'm not big on that point.

**Comment by andreas-bok-sociomantic** [comment]
Update based on comment from @ben-palmer-sociomantic 

```diff 
diff --git a/README.rst b/README.rst
index d15c9e8..175163d 100644
--- a/README.rst
+++ b/README.rst
@@ -100,8 +100,8 @@ refer to `compiling.rst`_ or build the documentation, cf. below.
 **Linking libdrizzle-redux**::
 
 Ensure the library is in your library and include paths. For releases prior to
-version ``v6.0.3`` linking your app against libdrizzle-redux requires the flag
-``-ldrizzle-redux6``::
+version ``v6.0.2`` [2]_ linking your app against libdrizzle-redux requires the flag
+``-ldrizzle-redux``::
 
     g++ app.c -oapp -ldrizzle-redux -lssl -lcrypto -pthread
 
@@ -122,6 +122,9 @@ dynamic library, e.g.::
 
     g++ app.c -oapp -I/usr/include/libdrizzle-redux6 -l:libdrizzle-redux6.so.13 -lssl -lcrypto -pthread
 
+.. [2] v6.0.2 added the major version to the package name and the library file
+       but the release is deprecated since the linking did not work correctly.
+
 **Building documentation**
 
 Documentation can be generated in several formats::
diff --git a/docs/compiling.rst b/docs/compiling.rst
index c4f3e9c..5fb285e 100644
--- a/docs/compiling.rst
+++ b/docs/compiling.rst
@@ -90,13 +90,13 @@ Linking Your Application
 ------------------------
 
 Ensure the library is in your library and include paths. For releases prior to
-version ``v6.0.3`` linking your app against libdrizzle-redux requires the flag
-``-ldrizzle-redux6``::
+version ``v6.0.2`` linking your app against libdrizzle-redux requires the flag
+``-ldrizzle-redux``::
 
     g++ app.c -oapp -ldrizzle-redux -lssl -lcrypto -pthread
 
 From version ``v6.0.3`` and later the API level of the library is appended to
-the installed library name. This is also reflected in the install path for
+the installed library name [1]_. This is also reflected in the install path for
 development headers which now follows the pattern::
 
     /<include-prefix>/libdrizzle-redux[MAJOR_VERSION]/libdrizzle-redux
@@ -115,3 +115,8 @@ dynamic library, e.g.::
 A tool called **libdrizzle-redux_config** is included to also assist with this.
 
 .. _RELEASE NOTES: https://github.com/sociomantic-tsunami/libdrizzle-redux/releases
+
+.. rubric:: Footnotes
+
+.. [1] v6.0.2 added the major version to the package name and the library file
+       but the release is deprecated since the linking did not work correctly.

```

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: And after the addition of the footnote?

**Review by mathias-lang-sociomantic — APPROVED** [review]
LGTM

**Comment by andreas-bok-sociomantic** [comment]
Tx

**Pull request merged**